privacyidea/lib/policy.py

```diff
@@ -2296,6 +2297,11 @@ def get_static_policy_definitions(scope=None):
                 'desc': _('If a user authenticates successfully reset the '
                           'failcounter of all of his tokens.')
             },
+            ACTION.ALWAYSINCREASEFAILCOUNTER: {
+                'type': 'bool',
+                'desc': _('If a user enters a correct pin or userstore password,'
+                          '  increases the failcounter of all of his tokens.')
```

Comment:
No, not all of his tokens.

Only these tokens, for which a challenge was triggered!

A user could have non-chal-resp tokens, and for those tokens there is no need to increase the failcounter!

Reply:
fixed in 4fe9f7f46e86b88dcd365672f86288d621345875.

Reply:
I think this still needs to be fixed?

"Increase the failcounter for all the tokens, for which a challenge has been triggered."

Reply:
fixed in 770f80617c7499699940c14e4b075104444da030.